playbooks/templates/rax-maas/lb_api_check_octavia.yaml.j2

```diff
@@ -0,0 +1,25 @@
+{% set label = "lb_api_check_octavia" %}
+{% set check_name = label+'--'+maas_lb_name %}
+type              : remote.http
+label             : "{{ check_name }}"
+period            : "{{ maas_check_period_override[label] | default(maas_check_period) }}"
+timeout           : "{{ maas_check_timeout_override[label] | default(maas_check_timeout) }}"
+disabled          : "{{ (inventory_hostname != groups['octavia_api'][0] or check_name | match(maas_excluded_checks_regex)) | ternary('true', 'false') }}"
```

Comment:
Is the group used here `octavia_api` or `octavia-api`? I noticed the group in the `maas-openstack-octavia.yml` playbook was calling `octavia-api` specifically.